functions/package.json

```diff
@@ -5,15 +5,19 @@
     "serve": "firebase emulators:start --only functions",
     "shell": "firebase functions:shell",
     "start": "npm run shell",
-    "deploy": "firebase deploy --only functions",
-    "logs": "firebase functions:log"
+    "logs": "firebase functions:log",
+    "build": "tsc",
+    "watch": "tsc --watch",
+    "deploy": "npm run build && firebase deploy --only functions",
+    "lint": "echo 'Linting is turned off, if you want to turn it on change this script to eslint '**/*.js''"
```

Comment:
It was a temporary workaround to avoid linting. It has now been setup to lint the ts files in ./src. (It is probably also worth noting that a lint ignore line was added so that it does not complain about using var when initializing JsPsychMetadata.)